Unaccountable-Magpie/WorkforceManagement
# Ag start

Purpose:
Adds the model for Computers controllers and views to reference
How it fits in context of the project:
Computers is required for the Employees controllers and views to function properly
Specific Feature affected:
Computers model 
How to test (Be thorough!):

1. checkout to my branch ag-start
2. start BangazonWebApp.sln from the terminal
3. check the ERD (shown in the top left corner of the link below) to make sure the keys are appropriately represented in the model
[https://files.slack.com/files-pri/T03F2SDTJ-FD1NF8LKC/image_from_ios.jpg](url)
4. run the program by clicking the green arrow IISExpress button and a browser should appear with a generic webpage
